Source/WebKit/Platform/IPC/Decoder.cpp

```diff
@@ -50,32 +54,63 @@ static MallocSpan<uint8_t> copyBuffer(std::span<const uint8_t> buffer)
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(Decoder);
 
-std::unique_ptr<Decoder> Decoder::create(std::span<const uint8_t> buffer, Vector<Attachment>&& attachments)
+std::unique_ptr<Decoder> Decoder::create(std::span<const uint8_t> buffer, Vector<Attachment>&& attachments
+#if OS(ANDROID)
+        , Vector<RefPtr<AHardwareBuffer>, 2>&& hardwareBuffers
```

Comment:
Why fiddle the hardwareBuffers thought static create methods, constructors etc.
Can't we just pass them after the `Decoder` creation `#if OS(ANDROID)`?